samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -1037,6 +1086,101 @@ To perform the DFU procedure, complete the following steps:
 
          .. include:: /includes/suit_fota_update_nrfcdm_test_steps.txt
 
+.. _fast_pair_locator_tag_testing_fw_update_notifications:
+
+Android notifications about firmware updates
+--------------------------------------------
+
+Testing steps for the firmware update notification feature require the `nRF Connect Device Manager`_ application on your test Android device.
+
+.. note::
+   The support for the Android notifications about the firmware updates in the context of the FMDN extension is a brand new feature.
+   Before you start the testing, ensure that versions for the following applications and services are equal or greater than the specified minimum versions:
+
+   * `Google Play Services`_ - ``v25.12.33``
+   * `nRF Connect Device Manager`_ - ``v2.5.0``
+
+.. note::
+   You can execute these testing steps in combination with the :ref:`fast_pair_locator_tag_testing_clock_sync` testing steps, as both test variants require you to wait for more than 24 hours.
+
+To test this feature, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: nRF52 and nRF53 DKs
+
+      1. Go to the :ref:`fast_pair_locator_tag_testing` section and follow the instructions on performing the FMDN provisioning operation.
+      #. Observe that **LED 3** is lit, which indicates that the device is provisioned as an FMDN beacon.
+      #. Power off the development kit and wait for 24 hours.
+      #. Power off your test Android device.
+      #. Power on the development kit.
+      #. |connect_terminal_specific|
+      #. Power on your test Android device and unlock it after the smartphone screen is turned on.
+      #. Wait for the Android device to connect to the development kit, read the local firmware version, and disconnect.
+      #. Observe that you get the Android notification about the new firmware update on your test Android device.
+
+         .. figure:: /images/bt_fast_pair_locator_tag_android_fw_update_notification.png
+            :scale: 80 %
```

Comment:
The image size needs to be reduced. Please follow the same size like the other .pngs in this document.